docs/tutorial/quickstart.md

```diff
@@ -12,16 +12,15 @@ Create a new Django project named `tutorial`, then start a new app called `quick
 
     # Create a virtualenv to isolate our package dependencies locally
     virtualenv env
-    source env/bin/activate  # On Windows use `env\Scripts\activate`
+    . env/bin/activate  # On Windows use `env\Scripts\activate`
```

Comment:
I'd argue that clarity is more important than portability here.
- not everyone is familiar with `.`, and it's more difficult to lookup than `source`
- might be confused with 'current directory'
- anyone using a shell that doesn't implement source will probably know to use `.`